docs/contribute/doc-release-process.md

```diff
@@ -0,0 +1,119 @@
+---
+description: Release process for documentation that uses a versioning system.
+sidebar_position: 8
+---
+
+# Documentation release process
```

Comment:
```suggestion
# Release a new docs version
```

Can we update the title of this page to start with a verb, to match most of the other pages in this section?

Reply:
This is a probably a good idea given the versioning audience is a subset of the whole